mods/cnc/languages/lua/en.ftl

```diff
@@ -0,0 +1,143 @@
+## campaign
+objective-failed = Objective failed
+objective-completed = Objective completed
+
+new-primary-objective = New primary objective
+new-secondary-objective = New secondary objective
+
+# cnc64gdi01
+destroy-sams = Destroy the SAM sites protecting the Obelisk.
+destroy-obelisk = Destroy the Obelisk.
+destroy-biotech = Destroy the biotech facility.
+
+## cnc64gdi01, gdi03, gdi05abc, gdi07 gdi08ab, gdi09
+destroy-sams = Destroy the SAM sites to receive air support.
```

Comment:
This is the same key as in line 9. I suppose line 9 should get a prefix to resolve the conflict?